src/fu-util.c

```diff
@@ -3484,6 +3524,16 @@ fu_util_security(FuUtilPrivate *priv, gchar **values, GError **error)
 	if (priv->no_unreported_check)
 		return TRUE;
 
+	/* any things we can fix? */
+	for (guint j = 0; j < attrs->len; j++) {
+		FwupdSecurityAttr *attr = g_ptr_array_index(attrs, j);
+		if (fwupd_security_attr_get_bios_attr_id(attr) != NULL &&
+		    fwupd_security_attr_get_bios_attr_value(attr) != NULL) {
+			if (!fu_util_security_modify_bios_attr(priv, attr, error))
+				return FALSE;
+		}
+	}
+
 	/* upload, with confirmation */
 	return fu_util_upload_security(priv, attrs, error);
```

Comment:
if `attrs->len > 0` and you made it this far (didn't say no) you should offer to reboot at the end of the func.

Reply:
Agreed, I've made it a bit more sensible.